src/geo_traits_impl/multi_line_string.rs

```diff
@@ -0,0 +1,63 @@
+use super::{LineStringType, MultiLineStringType};
+use bytemuck::TransparentWrapper;
+use geo_traits::{Dimensions, LineStringTrait};
+
+impl geo_traits::MultiLineStringTrait for MultiLineStringType {
+    type T = f64;
+    type LineStringType<'b>
+        = &'b LineStringType
+    where
+        Self: 'b;
+
+    fn num_line_strings(&self) -> usize {
+        self.0.len()
+    }
+
+    fn dim(&self) -> Dimensions {
+        self.line_string(0).unwrap().dim()
```

Comment:
here too

Reply:
Addressed in Addressed in https://github.com/georust/geojson/pull/245/commits/743ef4ea42d127cbeecdef536c5399b113e2d5f2a